Components/ListIcons/index.twig

```diff
@@ -15,7 +15,9 @@
         </div>
         {% endif %}
         <div class="item-linkWrapper">
-          <a href="{{ item.link.url }}" class="item-link button--link">{{ item.link.title }}</a>
+          <a href="{{ item.link.url }}" class="item-link button--link">
```

Comment:
While on it: could you please add the target?

Reply:
Oh, and please add `.button` to the classes, as I feel we shouldn't use a modifier without it's base class. (Yes, I'm aware this will change the appearance.)

Reply:
both done now!